src/main/webapp/WEB-INF/FoodInfo.csv

```diff
@@ -0,0 +1,43 @@
+Wheat & Rye (Bread),1.3
+Maize (Meal),1.2
+Barley (Beer),1.2
+Oatmeal,2.6
+Rice,3.7
+Potatoes,0.5
+Cassava,1.1
+Cane Sugar,3.2
+Beet Sugar,1.8
+Other Pulses,1.4
+Peas,0.8
+Nuts,1.3
+Groundnuts,3.3
+Soymilk,0.9
+Tofu,2.6
+Soybean Oil,3.9
+Palm Oil,7.2
+Sunflower Oil,3.5
+Rapeseed Oil,3.5
+Olive Oil,5.1
+Tomatoes,0.7
+Onions & Leeks,0.4
+Root Vegetables,0.4
+Brassicas,0.4
+Other Vegetables,0.4
+Citrus Fruit,0.3
+Bananas,0.8
+Apples,0.4
+Berries & Grapes,1.4
+Wine,1.6
+Other Fruit,0.7
+Coffee,8.2
+Dark Chocolate,5
+Bovine Meat (beef herd),60.4
+Bovine Meat (dairy herd),34.1
+Lamb & Mutton,40.6
+Pig Meat,10.6

```

Comment:
This is unrelated to code but this should maybe be `Pork` since I don't know many people who use the term `pig meat`.